README.md

```diff
@@ -1320,6 +1320,7 @@ Table of Contents
   * [Mindmup.com](https://www.mindmup.com/) — Unlimited mind maps for free, and store them in the cloud. Your mind maps are available everywhere, instantly, from any device.
   * [Mockplus iDoc](https://www.mockplus.com/idoc) - Mockplus iDoc is a powerful design collaboration & handoff tool. Free Plan includes 3 users and 5 projects with all features available.
   * [mockupmark.com](https://mockupmark.com/create/free) — Create realistic t-shirt and clothing mockups for social media and E-commerce, 40 free mockups.
+  * [Microsoft Designer](https://designer.microsoft.com/) - Stunning designs in a flash with Microsoft Designer.
```

Comment:
"join the waitlist...." this list is for services that are active and where pricing details are clear etc.

Thanks for contributing but think we'll pass on this one for now